device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/gearbox_config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "sesto-1",
+      "address": "0x1000",
+      "lib_name": "libsai_phy_sesto-1.so",
+      "firmware_path": "/tmp/phy-sesto-1.bin",
+      "config_file": "/usr/share/sonic/hwsku/sesto-1.json",
+      "sai_init_config_file": "/usr/share/sonic/hwsku/sesto-1.bcm",
```

Comment:
are these final path for the config files and firmware image?

Reply:
lib_name and firmware_path is vendor specific.
config_file and sai_init_config_file path  (/usr/share/sonic/hwsku/) is standard/generic.

Reply:
sesto-1.json/.bcm is vendor specific as well.